src/docs/product/data-management-settings/dynamic-sampling/benefits-dynamic-sampling.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Benefits of Dynamic Sampling
+sidebar_order: 25
+redirect_from:
+  - /product/sentry-basics/metrics/
+  - /product/sentry-basics/sampling/
+description: "Learn more about the benefits of dynamic sampling for your transactions."
+---
+
+<Note>
+
+When we talk about SDK sample rates on this page, we're only referring to your [transaction sample rate](/platform-redirect/?next=/configuration/sampling/#sampling-transaction-events) (or `tracesSampleRate`), not your [error sample rate](/platform-redirect/?next=/configuration/sampling/#sampling-error-events).
+
+</Note>
+
+_Sampling_ is a way to manage your application’s scale while ensuring you have the comprehensive data to manage the performance and health of your application. At low volumes, storing all your data is feasible and reasonable, but as your application scales, storing a raw copy of everything has diminishing returns. That’s where dynamic sampling comes in.
+
+Dynamic sampling is better than using your SDK to control how much data you send to Sentry. It’s difficult to decide what an appropriate client-side sample rate would be, and this can change over time. Instead, we recommend you send us 100% of your data and let us manage the sampling. Here are the benefits.
+
+## Automatic Sampling for High Quality Data
+
+If you’re using our [Dynamic Sampling](/product/data-management-settings/dynamic-sampling/) feature, Sentry automatically applies intelligent dynamic sampling to your data at higher volumes. This allows you to scale up affordably without having to micromanage your SDK sampling configuration. We’ll take care of picking valuable samples to retain on your behalf, while also giving you controls to express priorities that best fit your use case.
+
+Our dynamic sampling works best when you send us as much data as you can. The more a complete picture of your application we have, the better decisions we make on your behalf to retain data and detect and create new issues. If feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume we’ll adapt automatically. As a bonus your per-transaction pricing goes down accordingly.
```

Comment:
```suggestion
Our dynamic sampling works best when you send us as much data as you can. The more a complete picture of your application we have, the better decisions we make on your behalf to retain data and detect and create new issues. If feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly.
```